mpas_analysis/obs/observational_datasets.xml

```diff
@@ -468,25 +472,50 @@
       HadISST Nino 3.4 Index
     </name>
     <component>
-      ocean
+      ocean (for coupled simulations only)
```

Comment:
This needs to be `ocean`.  It is parsed to determine which table to put this observation into and it will be dropped from the table if it is not `ocean`.